src/sections/CareersHero/index.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import Section from "root/components/Section";
+import Navbar from "root/components/Navbar";
+import Typography from "root/components/Typography";
+import Wave from "../../assets/images/wave-careers-hero.inline.svg";
+
+import "./index.css";
+
+const CareersHero = () => (
+  <div styleName="root">
+    <Section verticalSpacing={false}>
+      <Navbar currentPage="careers" theme="primary" />
+      <div styleName="title">
```

Comment:
I suggest having `title-desktop` and `title-mobile` because we'll need to have a `Raising your growth` in the same `Typography` to mobile